src/papi.c

```diff
@@ -3246,7 +3248,12 @@ PAPI_read_ts( int EventSet, long long *values, long long *cycles )
  *	PAPI_accum adds the counters of the indicated event set into the array values. 
  *	The counters are zeroed and continue counting after the operation.
  *	Note the differences between PAPI_read and PAPI_accum, specifically 
- *	that PAPI_accum resets the values array to zero. 
+ *	that PAPI_accum adds the current counter values to the existing counter values
```

Comment:
Same comment as above:

Note that the values stored in the array don't have to be "existing counter values" but can be any arbitrary values provided by the user.

I suggest changing the wording to the following:

"Note the differences between PAPI_read() and PAPI_accum(). Specifically, PAPI_accum() adds the counter values to the values stored in the array (the second parameter in PAPI_accum()) and then resets the counters to zero."